### PR TITLE
Fix KernelAverager.get() so returns scalar, never 1d array

### PR DESCRIPTION
**Pull request by brews**
Fixes inconsistent behavior with `KernelAverager` instances getting ragged arrays because `self.get()` sometimes returns 1d arrays and sometimes returns scalars. This lead to deprecation warnings and slow projection speed in projection configurations using a Bartlett running average.

`KernelAverager.get()` should now always return a scalar.

Also:

- `MedianAverager.get()`
- `MeanAverager.get()`

### CODE REVIEW

**Comment by andyhultgren** [comment]
"This lead to deprecation warnings and slow projection speed in projection
configurations using a Bartlett running average."

nice!!

**Comment by brews** [comment]
~I'm testing this PR against the full test suite downstream in `impact-calculations`. Don't merge until this passes.~

Edit: Okay, everything passes. We're good.

**Comment by jrising** [comment]
Looks fine! I do notice, however, that the object given back by `np.mean` and `np.median` is also of type `numpy.float64`, like this is before you call `.item`. Since `impact-common` is a bit of a nuisance to update (since-- I think-- it's built into the singularity container), can you plug `.item` calls on the other classes' `get` methods here too?

**Comment by brews** [comment]
`MeanAverager.get()` now has `.item()` called on its output it before its returned. This ensures that anything returned is a scalar and not an array, which I think is what you're looking for @jrising.

The other `.get()` method for `BucketAverager` appears to operate on scalars, so no need to `.item()`.

I also found an identical, duplicate class definition for `MeanAverager`. I removed it so there is now only one `MeanAverager` definition to rule them all.

**Edit**: Nevermind, one was `MedianAverager` the other was `MeanAverager` I need to drink my coffee before I code. I'll fix this.

**Comment by brews** [comment]
I'm rerunning system tests on the changes again and then I'll merge if there is no issue.

CI/CD with travis is screwed up because of changes in Travis (it needs to get dropped for github actions). Travis is no longer a reliable tester.

**Comment by brews** [comment]
The full test suite downstream in `impact-calculations` passes with these changes.

**Review by jrising — APPROVED** [review]
Great. Thanks for these further changes.

**Pull request merged**
